### DISCUSSION TITLE
Instable connection (keep disconnecting) on a long distance connection (Linux Wayland to Mac OS X)

### DISCUSSION THREAD

**Discussion by martinaware** [rustdesk/rustdesk#8355] [General]
### Bug Description

I tried to connect to a Mac (Mac Book Pro / Mac OS X) from Linux (Debian / XFCE / Wayland).
It was a long distance connection, from France to Turkey.
The connection was mostly unstable, and kept asking for password.
Is it a distance issue ? A bandwith issue ?
Is there any way to get around ?
Sadly, I had to switch to teamviewer, which did work well.

### How to Reproduce

Get access to a computer with a bandwith a bit slow. (but good enough to make skype calls)
I guess this is reproductible with a long distance connection.
Maybe the OS is important (Linux to Mac OS X).

### Expected Behavior

Rustdesk keep disconnecting, generating a new password and asking for the new password.

### Operating system(s) on local side and remote side

Debian 12 (XFCE) -> Mac OS X

### RustDesk Version(s) on local side and remote side

1.2.3 - > 1.2.3

### Screenshots

No screenshots

**Comment by rustdesk** [comment]
https://www.reddit.com/r/rustdesk/comments/1cr8kfv/should_you_selfhost_a_rustdesk_server/

**Reply by martinaware** [reply]
Thanks, I will try to install it with docker.
